src/components/atoms/Button/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+import loginIcon from './login-icon.svg'
+import styles from './styles.css'
+
+function Button(props) {
+  return (
+    <button className="button" onClick={props.onclick}>
+      <img className="buttonIcon" src={loginIcon} alt="button-icon" />
```

Comment:
there's no need to use the label `buttonIcon`, this is a scoped component, you can only call this class as `icon`. Also the `alt` should contain a valuable message, you can put the `props.text` inside of it.